CircleandCross2/Program.cs

```diff
@@ -0,0 +1,160 @@
+﻿using System;
+
+namespace CircleandCross2
+{
+	class Program
+	{
+
+		static void Main(string[] args)
+		{
+			try
+			{
+
+
+				Console.WriteLine("Tic Tac Toe");
+
+				string[,] array = new string[3, 3] { { "[ ]", "[ ]", "[ ]" }, { "[ ]", "[ ]", "[ ]" }, { "[ ]", "[ ]", "[ ]" } };
+
+				for (int i = 0; i < array.GetLength(0); i++)
+				{
+					for (int j = 0; j < array.GetLength(1); j++)
+					{
+						string arr = array[i, j];
+						Console.Write(arr);
+					}
+					Console.WriteLine();
+				}
+
+
+				for (int a = 1; a <= 5; a++)
+				{
+
+					Console.WriteLine("podaj pozycje 'o' w lini 'i'");
+					string tekst = Console.ReadLine();
+					int pozycja1i = int.Parse(tekst);
```

Comment:
Check if numbers are in the array range.

Reply:
i changed it but not all works like i want

Reply:
i do it